### DISCUSSION TITLE
Now GPU fans make a bad noise when they start spinning

### DISCUSSION THREAD

**Discussion by zSuperMicrowave** [Rem0o/FanControl.Releases#1906] [Q&A] [answered]
I installed the program because i was tired of hearing the GPU fans suddently start spinning every time the temperature got even a tiny bit higher. During configuration i chose to configure the GPU fans and during the configuration I heard a bad noise of vibrations, so i pressed on cancel because I was scared of braking something. Now my fan always makes that noise when it starts to spin, is it a coincidence  or could it be due to the software? Can it be solved applying a curve or something to the fan?

**Comment by Rem0o** [comment, marked as answer]
My guess is you never ran your GPU fan at the % FanControl made it run at. During config it does a full sweep, 0 to almost 100%  If you run your fan at the threshold at which it starts spinning, you might have this behavior. 

You should apply a fan curve where the fan will be comfortably spinning ( > 500-600 rpm ).

**Reply by zSuperMicrowave** [reply]
Yeah I think that's the problem. I always go in panic when I hear new sounds from my pc.

I've now got a curve that always makes the fan spin. Now it can't go below about 660 rpms.
